.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: generic
+
+os: osx
+osx_image: xcode6.4
+
+branches:
+  only:
+  - master
+
+env:
+  - PYVER="27"
+  - PYVER="35"
+  - PYVER="36"
+
+before_install:
+  - curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall > ~/uninstall_homebrew
+  - /usr/bin/ruby ~/uninstall_homebrew -fq
+  - rm ~/uninstall_homebrew
```

Comment:
Homebrew tends to muck with `conda`. So it is better to remove it entirely before proceeding.